index.org

```diff
@@ -299,7 +300,7 @@ This ~(task map, reset timestamp, trace clock frequency)~ tuple constitutes the
 If the input buffer of the serial device is filled, packets will be lost or corrupted.
 A warning will be printed before this buffer is overflowed, or if the buffer size cannot be determined.
 
-** RTIC application constrains
+** RTIC application constraints
```

Comment:
I don't think it is quite clear why this is a constraint. What is the desired functionality and why is the macro definition not quite ideal?

Reply:
What constraint do you refer to?